examples/jsm/renderers/CSS3DRenderer.js

```diff
@@ -246,7 +246,12 @@ class CSS3DRenderer {
 					if ( object.rotation2D !== 0 ) _matrix.multiply( _matrix2.makeRotationZ( object.rotation2D ) );
 
 					_matrix.copyPosition( object.matrixWorld );
-					_matrix.scale( object.scale );
```

Comment:
Maybe better to [decompose](https://threejs.org/docs/index.html?q=matrix#api/en/math/Matrix4.decompose) the world matrix and then use the resulting scale and position for setting up `_matrix`.